Spreadsheet/style.css

```diff
@@ -74,29 +73,26 @@ header {
     overflow: scroll;
 }
 
-.box {
+.cell {
     border: 1px solid #01A9B4;
     width: 100px;
     height: 30px;
     text-align: center;
     line-height: 30px;
 }
 
-.box-border {
+.selectedCell {
```

Comment:
same as `numbers`, here `box-border` doesn't describe what the element is or what its state is but rather its style. Prefer a name that represents concepts rather than content/shape. Here for example, this class is applied to cells that are selected, so `selectedCell` seems like a much more descriptive name for it.